tests/state_testing/test_state.py

```diff
@@ -31,7 +31,7 @@ def _node_ID_str(num):
     return "v{}".format(num + 1)
 
 
-@settings(max_examples=200, deadline=None)
+@settings(max_examples=250, deadline=None)
```

Comment:
why?

Reply:
Because I added a new rule, so I want the state machine to be able to explore similar depths of states despite that there is now more breadth to consider